src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseService.cs

```diff
@@ -434,14 +435,23 @@ public IEnumerable<Guid> GetReferencedReleaseFileVersions(Guid releaseId, params
         private async Task<Either<ActionResult, bool>> ValidateReleaseSlugUniqueToPublication(string slug,
             Guid publicationId, Guid? releaseId = null)
         {
-            if (await _context.Releases.AnyAsync(r => r.Slug == slug && r.PublicationId == publicationId && r.Id != releaseId))
+            var releases = await _context.Releases
+                .Include(r => r.Publication)
```

Comment:
Do we need to include the publication if we're not using it?

Reply:
Yeah good spot - Thanks - Done